packages/database/src/migrations/20201116013159-AddMinMaxValuestoMOSReport-modifies-data.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const OLD_CHART_CONFIG = {
+  "UNFPA_RH_MOS_Condoms_male": {
```

Comment:
It doesn't really matter, but one of the advantages of putting this into an object is that you can remove the quotes around keys, which your linter should be able to do automatically.